jenkins/bootstrap_test.py

```diff
@@ -1200,15 +1200,15 @@ def Check(job, name):
         self.CheckBootstrapYaml('job-configs/kubernetes-jenkins-pull/bootstrap-maintenance-pull.yaml', Check)
 
     def testBootstrapPullYaml(self):
-        is_modern = lambda name: 'unit' in name or 'verify' in name
+        is_modern = lambda n: any(w in n for w in ['unit', 'verify', 'node'])
         def Check(job, name):
             job_name = 'pull-%s' % name
             self.assertIn('max-total', job)
             self.assertIn('repo-name', job)
             self.assertIn('.', job['repo-name'])  # Has domain
             self.assertIn('timeout', job)
             self.assertIn('json', job)
-            modern = is_modern(name)
+            modern = is_modern(name) and 1 or 0
```

Comment:
I think `is_modern` returns True/False rather than an obj as `re.match` does? If the problem is you cannot compare True/False with 1/0 then this is fine.